lambda/src/main/java/org/dcsa/conformance/lambda/LambdaToolkit.java

```diff
@@ -5,14 +5,18 @@
 import com.amazonaws.services.lambda.model.InvokeRequest;
 import com.fasterxml.jackson.databind.JsonNode;
 import java.util.function.Consumer;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
 import org.dcsa.conformance.sandbox.state.ConformancePersistenceProvider;
 import org.dcsa.conformance.sandbox.state.DynamoDbSortedPartitionsLockingMap;
 import org.dcsa.conformance.sandbox.state.DynamoDbSortedPartitionsNonLockingMap;
+import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
 
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class LambdaToolkit {
   public static ConformancePersistenceProvider createPersistenceProvider() {
-    DynamoDbClient dynamoDbClient = DynamoDbClient.builder().build();
+    DynamoDbClient dynamoDbClient = DynamoDbClient.builder().region(Region.EU_NORTH_1).build();
```

Comment:
I got the knowledge from Sonar on this one:
"If the region is not specified when creating a new AwsClient with an AwsClientBuilder, the AWS SDK will execute some logic to identify the endpoint automatically.
While it will probably identify the correct one, this extra logic will slow down startup time, already known to be a hotspot.
You should therefore always define the logic to set the region yourself. This is typically done by retrieving the region from the Lambda provided AWS_REGION environment variable.
This will make the code more explicit and spare initialization time.
This rule reports an issue when the region is not set when creating an AwsClient."